.github/workflows/build-binaries.yml

```diff
@@ -43,24 +43,19 @@ jobs:
         uses: Swatinem/rust-cache@v2
         with:
           workspaces: temporalio/bridge -> target
-
-      # Prepare
-      # Using fixed Poetry version until
-      # https://github.com/python-poetry/poetry/issues/7611 and
-      # https://github.com/python-poetry/poetry/pull/7694 are fixed
-      - run: python -m pip install --upgrade wheel "poetry==1.3.2" poethepoet
-      - run: poetry install --no-root --all-extras
+      - uses: astral-sh/setup-uv@v5
+      - run: uv sync --all-extras
+      - run: uv build
 
       # Add the source dist only for Linux x64 for now
       - if: ${{ matrix.package-suffix == 'linux-amd64' }}
-        run: poetry build --format sdist
+        run: uv build --sdist
 
-      # Build and fix the wheel
-      - run: poetry run cibuildwheel --output-dir dist
-      - run: poe fix-wheel
+      # Build the wheel
+      - run: uv run cibuildwheel --output-dir dist
 
-      # Simple test
-      - run: poe test-dist-single
+      # Run a test
+      - run: uv run pytest -k test_activity_hello
```

Comment:
The goal of the original form of this was to do a literal `pip install` of the built wheel to confirm it works on the platform. Can we do some form of `pip install` on this step? Maybe in a completely new folder as a smoke test? Could even be a separate job with its own matrix. Just want to confirm the built wheel works on every platform.

Reply:
Thanks, I've modified it to be like
```yaml
      - name: Test wheel
        run: |
          python -m venv .test-venv
          ./.test-venv/bin/pip install 'protobuf>=3.20'
          ./.test-venv/bin/pip install --no-index --find-links=./dist temporalio
          ./.test-venv/bin/pip install pytest
          ./.test-venv/bin/python -m pytest -k tests/test_activity_hello
          rm -r .test-venv
```